test/Datadog.Trace.ClrProfiler.IntegrationTests/DbCommandTests.cs

```diff
@@ -29,15 +26,14 @@ public void SubmitsTraces(string packageVersion)
 
```

Comment:
I think you're going to run into issues here because the `Samples.DbCommand` will not be built multiple times (only `Samples.SqlCommand` will, with your changes to `PackageVersionsGeneratorDefinitions.json`). I think you'll need to enter a duplicate record there so that the project gets built for each NuGet version, or perhaps extend the syntax of the `SampleProjectName` to allow semi-colon delimited project names so we build multiple projects per NuGet package version.

Reply:
These are meant to be completely separate tests/samples. I regenerated `PackageVersionsGeneratorDefinitions.json` in 4c278b9174e4202c91736dbc0a2f26d13b41776e.